src/Widgets/CardWidget/CardWidgetComponent.tsx

```diff
@@ -30,13 +32,17 @@ provideComponent(CardWidget, ({ widget }) => {
   if (widget.get('cardExtended')) cardClassNames.push('card-extended')
 
   return (
-    <WidgetTag className={cardClassNames.join(' ')}>
+    <WidgetTag
+      className={cardClassNames.join(' ')}
+      onClick={(e) => togglePlayPauseRef.current?.togglePlayPause(e)}
+    >
```

Comment:
* [ ] FYI Having this logic in all parent components has a certain complexity. Since we handle clicks anywhere, how about using `onMouseUp` instead of `onClick`? The mouse events bubble by default.

Reply:
I see. How about making ImageOrVideo a wrapper component? So instead of
```tsx
<WidgetTag>
  <ImageOrVideo />
  { stuff }
</WidgetTag>
```

we'd go with the event bubbling version:
```tsx
<WidgetTag>
  <ImageOrVideo>
    { stuff }
  </ImageOrVideo>
</WidgetTag>
```
We could rename it to `<BackgroundImageOrVideo>`.
We need an additional wrapper, but this can correctly handle click events on children.
If the added wrapper causes CSS issues, we could try it with `display: contents;`.

Reply:
I gave it a try, but there is quite a lot of css that assumes, that the background image/video is a direct child of the card. I would leave it as is for now.